examples/functional-iteration/functional_iteration.description

```diff
@@ -1,2 +1,2 @@
-// Iterable operations are a set of built-in operations that are available on
+// Iterable operations are a set of built-in operations that are available for
```

Comment:
You perform an iterable operation on a particular value. Wouldn't "on" be the better choice here?

Reply:
Since we are talking about the available operations here and not about performing an operation, "for" is more applicable.